packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -373,6 +318,22 @@ const ProfileMenu = (props: Props): any => {
                 {t('user:usermenu.profile.inviteCode')}: {selfUser.inviteCode.value}
               </h2>
             )}
+            {selfUser?.userRole.value !== 'guest' && (
```

Comment:
You merged the code wrong here... the above code was written by me and merged into dev... makes me wonder what else is merged wrong...